<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DHCP Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="layout_right">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_interface">
            <property name="text">
             <string>网卡</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cb_interface"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_router">
            <property name="text">
             <string>服务器IP</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="server_edit">
            <property name="text">
             <string>192.168.1.1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>DHCP 选项</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="ip_pool_label">
               <property name="text">
                <string>DHCP地址地址池</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="ip_pool_edit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="pool_size_label">
               <property name="text">
                <string>地址池大小</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="pool_size_edit">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lease_label">
               <property name="text">
                <string>租期</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lease_edit">
               <property name="text">
                <string>2880</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="router_label">
               <property name="text">
                <string>网关</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="router_edit">
               <property name="text">
                <string>192.168.1.1</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="mask_label">
               <property name="text">
                <string>子网掩码</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="mask_edit">
               <property name="text">
                <string>255.255.255.0</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="dns_label">
               <property name="text">
                <string>DNS</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="dns_edit">
               <property name="text">
                <string>8.8.8.8</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QRadioButton" name="vdi_radio">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>VDI</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="voi_radio">
            <property name="text">
             <string>VOI</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="version_label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;v1.3  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="statu_label">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/images/offline-point.png);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="run_btn">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Open Config</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save </string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cb_interface</tabstop>
  <tabstop>server_edit</tabstop>
  <tabstop>ip_pool_edit</tabstop>
  <tabstop>pool_size_edit</tabstop>
  <tabstop>lease_edit</tabstop>
  <tabstop>router_edit</tabstop>
  <tabstop>mask_edit</tabstop>
  <tabstop>dns_edit</tabstop>
  <tabstop>run_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
